docs/en/architecture.md

```diff
@@ -0,0 +1,22 @@
+Deploying this solution with the default parameters builds the following environment in the AWS Cloud.
+
+![architecture](./images/arch.png)
+*Figure 1: AI Video Super Resolution architecture*
+
+This solution deploys the Amazon CloudFormation template in your Amazon Cloud Technology account and completes the following settings.
+
+1. Use [Amazon API Gateway][api-gateway] to implement user access interface.
+1. The [Amazon Lambda][lambda] function is used to receive user requests and start the calculation job of [Amazon Batch][Batch].
+1. [Amazon S3][s3] is used for persistent video storage.
+1. [Amazon EFS][efs] is used for temporary storage of intermediate files during video processing.
+1. [Amazon Batch][Batch] is used to pull pre-built model images from ECR, and start [Amazon EC2 Inf1][inf1] instances through ECS to run calculation jobs, and perform video slicing, sharding, and merging operations.
```

Comment:
a new proposal to highlight the batch is,

Batch schedules the super resolution jobs(including video slicing, sharding and merging operations) on pre-defined computing resources with scaling, it automatically releases the backend computing resource after all jobs are done.